RecoLocalTracker/SiPixelRecHits/src/PixelCPEGeneric.cc

```diff
@@ -518,46 +570,147 @@ float PixelCPEGeneric::generic_position_formula(int size,                    //!
 //!  and the inner cluster charge, projected in x and y.
 //-----------------------------------------------------------------------------
 void PixelCPEGeneric::collect_edge_charges(ClusterParam& theClusterParamBase,  //!< input, the cluster
-                                           int& Q_f_X,                         //!< output, Q first  in X
-                                           int& Q_l_X,                         //!< output, Q last   in X
-                                           int& Q_f_Y,                         //!< output, Q first  in Y
-                                           int& Q_l_Y                          //!< output, Q last   in Y
-                                           ) const {
+                                           bool& del_Xmin,
```

Comment:
none of these & are useful given bools/ints

Reply:
I think this is meant as a setter for those values? But if not, agreed